grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -440,6 +440,11 @@ class UrlMappings {
         }
 
         // Invoice API
+        "/api/invoices/items"(parseRequest: true) {
+            controller = "invoiceApi"
+            action = [GET: "getAllInvoiceItems"]
+        }
+
```

Comment:
I don't mind it, but I think we used to do it via the export mapping, with a proper param (e.g. "orderItems").

Reply:
yes this is probably the way to go, this should be on `/api/invoiceItems/`